### PR TITLE
Bugfix/output directories

### PR DESCRIPTION
**Pull request by Vkale1**
Some minor modifications to output directory.

- genomad_out was a nested directory
- ice finder tmp takes up a lot of space and not needed. This is the best solution I could find.. happy if you find a better one!

### CODE REVIEW

**Review by mberacochea — CHANGES_REQUESTED** [review]
Thanks Varsha.. I left a comment with an alternative approach to the deletion of that pesky tmp folder

File: main.nf [hunk 1]
```diff
@@ -95,6 +96,12 @@ workflow {
 		file( "${params.outdir}/prediction/icefinder_results/tmp"),
 		file( "${params.outdir}/prediction/icefinder_results/result")
 	)
+    // remove ICEFINDER tmp directory
+    CLEANUP(
+        file( "${params.outdir}/prediction/icefinder_results/tmp"),
+        ICEFINDER.out.icf_summ_files
+    )
```
[the remark this answers is not in the document]
Comment (Vkale1) [reply main.nf:103]: So I tried this I think.. the problem is deleting the bath bound to the container or without specifying the absolute path doesn't work. It deleted essentially the symlink of the staged directory. Hence the use of "readlink" I can try and get the absolute path within this folder. But I just worried if it randomly failed it wouldn't be restartable - hence put it totally external of ice finder, but still running only after ice finder.

If you are able to get this to work though do let me know!

[the remark this answers is not in the document]
Comment (mberacochea) [reply main.nf:103]: OK. Then another approach would be to copy the results (but that tmp folder) to another place (in Icefinder) and use that to be published (ping me on slack if this doesn't make sense)

[the remark this answers is not in the document]
Comment (Vkale1) [reply main.nf:103]: Note here for future reference- spoken in person. This is the best solution for now until we modify the ice finder container.

**Pull request merged**